vars/mavenCanaryRelease.groovy

```diff
@@ -22,7 +22,11 @@ def call(body) {
     sh "git checkout -b ${env.JOB_NAME}-${config.version}"
 
     if (autoUpdateFMP) {
-        patchFMPVersion()
+        try {
+            patchFMPVersion()
+        } catch (err) {
+            println "FMP patching failed due to ${err.message}"
```

Comment:
shouldn't this be `echo`

Reply:
It's better to use `println()` than. Better to leverage JVM construct than platform specific construct